src/garage/sampler/is_sampler.py

```diff
@@ -54,31 +56,58 @@ def __init__(self,
         self._hist = []
         self._is_itr = init_is
 
-        super().__init__(algo, env)
+        self.algo = algo
+        self.env = env
 
     @property
     def history(self):
-        """
+        """History of policies.
```

Comment:
Properties should be documented like attributes, not functions. See https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html for an example.